cra-rxjs-styled-components/src/components/repo-issues/Issues/Issue.style.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
```

Comment:
Rename this file to `Issues.view.styles.tsx` because these styles are for the `IssuesView` component.

Also, go through your styles file and correct the name. its `.styles.tsx` not `.style.tsx`